README.md

```diff
@@ -60,12 +60,33 @@ Project has no opportunity to authorize new users as edX's instance student.
 So you need to do this manually:
 1. Go to admin panel, then to `Edx telegram users` model (table).
 2. Make relationship between Telegram user and edX user in field `student`.
-3. Add telegram `id` and telegram `nickname`.
+3. Add telegram `id` and telegram `username` (optional move, it can be empty - `''`).
 
-If you don`t know telegram id and nickname, just go ahead and log this, i.e:
+If you don't know telegram id and nickname, just go ahead and log it there
+https://github.com/raccoongang/edx_telegram_bot/blob/master/edx_telegram_bot/edx_telegram_bot.py#L169:
 ```
 def help_command(self, bot, update):
     print update.message
```

Comment:
It should be mentioned that `print` won't be logged if the app is running under supervisor.